apps/ehr/tests/e2e/page/in-person/MedicationsPage.ts

```diff
@@ -0,0 +1,210 @@
+import { expect, Page } from '@playwright/test';
+import { dataTestIds } from '../../../../src/constants/data-test-ids';
+import { Dialog, expectDialog } from '../patient-information/Dialog';
+import { SideMenu } from '../SideMenu';
+import { EditNoteDialog, expectEditNoteDialog } from './EditNoteDialog';
+
+export class MedicationsPage {
+  #page: Page;
+
+  constructor(page: Page) {
+    this.#page = page;
+  }
+  sideMenu(): SideMenu {
+    return new SideMenu(this.#page);
+  }
+
+  async selectMedication(medication: string): Promise<void> {
+    await this.#page
+      .getByTestId(dataTestIds.telemedEhrFlow.hpiCurrentMedicationsInput)
+      .locator('input')
+      .fill(medication);
+    await this.#page.locator('li').getByText(medication, { exact: false }).click();
+  }
+
+  async verifyScheduledMedication(medicationName: string, dose: string): Promise<void> {
+    await expect(
+      this.#page
+        .getByTestId(dataTestIds.telemedEhrFlow.hpiCurrentMedicationsScheduledList)
+        .filter({ hasText: medicationName + '(' + dose })
+    ).toBeVisible();
+  }
+  async verifyRemovedScheduledMedicationIsNotVisible(medicationName: string, dose: string): Promise<void> {
+    await expect(
+      this.#page
+        .getByTestId(dataTestIds.telemedEhrFlow.hpiCurrentMedicationsScheduledList)
+        .filter({ hasText: medicationName + '(' + dose })
+    ).not.toBeVisible();
+  }
+
+  async verifyAsNeddedMedication(medicationName: string, dose: string): Promise<void> {
```

Comment:
```suggestion
  async verifyAsNedeedMedication(medicationName: string, dose: string): Promise<void> {
```

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
oops i made a typo myself, how ironic